### ISSUE TITLE
Resonite data import tool

### ISSUE THREAD

**Issue by jae1911** [Yellow-Dog-Man/Resonite-Issues#3082]
### Is your feature request related to a problem? Please describe.

While writing [the wiki page about account deletion](https://wiki.resonite.com/Account_deletion), I noticed there are ways to download data from Resonite either using the GUI or CLI program provided, but no way to re-import it easily.

This could be an issue in the case of an user deleting their account, but wanting to come back later and restore their downloaded data.

### Describe the solution you'd like

Have a program to import downloaded data to an account.

This could work in a similar way as the account migration.

### Describe alternatives you've considered

Re-importing the data manually, though very tedious.

### Additional Context

- https://wiki.resonite.com/Account_deletion
- https://wiki.resonite.com/Account_data_download

### Requesters

U-j4 | j4.lc (Discord)

**Comment by Frooxius** [comment]
We can add something like this, but there's some caveats to this - when the account is deleted, it's gone. That cannot be ever used again anymore.

You could potentially reupload a lot of the data, but there *will* be some breakage. E.g. all the UserID will refer to the deleted account. The account migration translates some of this, which can help, but it won't be perfect.

You won't be to fully "come back later" from a deleted account - this is permanent.

**Comment by jae1911** [comment]
> You won't be to fully "come back later" from a deleted account - this is permanent.

Yes, this issue was opened so users have an easy option to at least regain part of their inventories if they decide to create a new account and start using Resonite again. (provided they downloaded their data using the options listed on the wiki, I'm aware deleted data can't just reappear magically)

**Comment by Frooxius** [comment]
Yeah that's fine! I know @ProbablePrime has been working on a such tool, but I'm not fully up to date with the state of it and how much it would take to finish.

That'll probably determine on how big of a task this is and when we should prioritize something like this (upvoting helps).

**Comment by shiftyscales** [comment]
Seeking evaluation from @ProbablePrime regarding the size / scope of this issue.
